tutorials/2.5/graphics-transform.php

```diff
@@ -160,7 +160,7 @@
 sf::Transform t4 = t1 * t2 * t3;
 </code></pre>
 <p>
-    You can apply several predefined transformations to the same transform as well. They will all be combined sequentially:
+    You can apply several predefined transformations to the same transform as well. They will all be combined sequentially, in reverse order: the last operation (here <code>scale</code>) is applied first, and will be affected by operations above it in code (second would be <code>translate(-10.f, 50.f)</code>, for example).
```

Comment:
Is it always in reverse order or just reverse order for the listed provided here?

Personally, I still don't understand why it is applied in reverse. 🤔

Reply:
Simple example: you start with a rectangle (black) as shown in the left graphic. The vector `v` represents each vertex' position -- the transformed rectangle is a result of transforming each vertex.
* First, you rotate `v` using the rotation matrix `R`. The result (blue) is `R * v`, which we call `w`.
* Now, you apply a translation `T` to `w`.  
Same as before, the result (green) is `T * w`, which is `T * (R * v)`.  
Matrix multiplication is associative; this is the same as `T * R * v`.
* Applying an additional scale transform `S` will yield `S * T * R * v` (red).

![grafik](https://user-images.githubusercontent.com/708488/70565136-c48cef80-1b91-11ea-990e-9d855601ca31.png)

You see that even though the first transform was `R`, it's the rightmost expression in the equation -- or the last statement in code. In other words, `R` is written closest to `v`, and `R * v` can be understood as an intermediate vector, which is then further transformed by `T`, and last `S`.

Reply:
I wonder if this topic (order of transform) is something that we should elaborate in the tutorial, or if it's out of scope.

Any other feedback to the PR?